bin/resty

```diff
@@ -62,11 +62,13 @@ GetOptions("c=i",             \(my $conns_num),
            "nginx=s",         \(my $nginx_path),
            "valgrind",        \(my $use_valgrind),
            "valgrind-opts=s", \(my $valgrind_opts),
+           "ns=s",            \@ns,
            "resolve-ipv6",    \(my $resolve_ipv6),
            "V|v",             \(my $version))
    or usage(1);
 
 check_option_format('--shdict', qr/^[_a-z]+\s+\d+(?i)[km]?\z/, 'NAME SIZE', \@shdicts);
+check_option_format('--ns',     qr/^(?:\d{1,3}(?:\.\d{1,3}){3}|\[[0-9a-fA-F:]+\])$/x, 'IP', \@ns);
```

Comment:
Will you add spaces to the regex (as I did in one of my previous comments) to make it more readable? We are using `/x` here already, which means spaces in the regex are not significant (except those inside a character class). Also this line seems too long, better break it up.